Reporting_SSIS/Reporting_SSIS/ReportingDB_M.conmgr

```diff
@@ -3,6 +3,10 @@
   DTS:ObjectName="ReportingDB_M"
   DTS:DTSID="{D975BEA8-FF35-4DB6-B172-3C44A2294352}"
   DTS:CreationName="OLEDB">
+  <DTS:PropertyExpression
+    DTS:Name="InitialCatalog">@[$Project::DestinationDBName]</DTS:PropertyExpression>
+  <DTS:PropertyExpression
+    DTS:Name="ServerName">@[$Project::ServerName]</DTS:PropertyExpression>
```

Comment:
@Sreenitha-Sampath As we have source and destination servers, rename ServerName to DestinationServerName